### PR TITLE
Add workflows for stolostron-engine

### PR DESCRIPTION
**Pull request by joeg-pro**
This PR adds some test workflows for use in builds of stolostron-engine.

### CODE REVIEW

File: ci-operator/step-registry/stolostron-engine/ci/OWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+../OWNERS
```
Comment (smg247) [inline ci-operator/step-registry/stolostron-engine/ci/OWNERS:1]: These symlinks aren't necessary. The owners files will cascade to child directories.

**Review by smg247 — COMMENTED** [review]
/approve

File: ci-operator/step-registry/stolostron-engine/ci/OWNERS (hunk 1 above)
Comment (joeg-pro) [reply ci-operator/step-registry/stolostron-engine/ci/OWNERS]: Ok.  Removed.

**Pull request merged**